packages/twenty-ui/src/display/chip/components/EntityChip.tsx

```diff
@@ -72,9 +63,50 @@ export const EntityChip = ({
         )
       }
       clickable={!!linkToEntity}
-      onClick={handleLinkClick}
       className={className}
       maxWidth={maxWidth}
     />
   );
 };
+
+export const EntityChip = ({
+  linkToEntity,
+  entityId,
+  name,
+  avatarUrl,
+  avatarType = 'rounded',
+  variant = EntityChipVariant.Regular,
+  LeftIcon,
+  className,
+  maxWidth,
+}: EntityChipProps) => {
+  if (isNonEmptyString(linkToEntity)) {
+    return (
+      <a href={linkToEntity}>
+        <InnerEntityChip
+          entityId={entityId}
+          name={name}
+          avatarUrl={avatarUrl}
+          avatarType={avatarType}
+          variant={variant}
+          LeftIcon={LeftIcon}
+          className={className}
+          maxWidth={maxWidth}
+        />
+      </a>
+    );
+  } else {
+    return (
```

Comment:
This would be nicer:
```
  const InnerEntityChipComponent = (
    <InnerEntityChip
      entityId={entityId}
      name={name}
      avatarUrl={avatarUrl}
      avatarType={avatarType}
      variant={variant}
      LeftIcon={LeftIcon}
      className={className}
      maxWidth={maxWidth}
    />
  );

  return linkToEntity ? (
    <a href={linkToEntity}>{InnerEntityChipComponent}</a>
  ) : (
    InnerEntityChipComponent
  );
```